pylibui/libui/box.py

```diff
@@ -54,13 +54,13 @@ def uiBoxDelete(box, index):
 # - int uiBoxPadded(uiBox *b);
 def uiBoxPadded(box):
     """
-    Sets the padding of the box.
+    Returns whether the box is padded.
 
     :param box: uiBox
     :return: int
```

Comment:
No, everything inside pylibui/libui should be almost similar to the C lib, just a thin wrapper. So, if the C function says:

# - int uiBoxPadded(uiBox *b);

as you can read in the signature before the python function, it means that it gets a uiBox and returns an int, so we must respect that interface.